docker-compose.yml

```
@@ -0,0 +1,38 @@
+# For locally developing the add-on as a standalone Docker container 
+version: '3.8'
+
+services:
+  asterisk:
+    image: ghcr.io/tech7fox/amd64-addon-asterisk
+    build:
+      context: ./asterisk
+      network: host
+    tty: true
+    entrypoint: /init
+    init: false
+    environment:
+      # To connect to the HA API
+      HOME_ASSISTANT_URL: http://home-assistant:8123
+      HOME_ASSISTANT_TOKEN: <my_secret_token>
+    volumes:
+      # Mount configuration read by the addon
+      - ./asterisk/config.json:/data/options.json:ro
+      
+      # To avoid having to rebuild the whole image, mount the init files inside it
+      # - ./asterisk/rootfs/etc/cont-init.d/asterisk.sh:/etc/cont-init.d/asterisk.sh:ro
+      # - ./asterisk/rootfs/etc/cont-init.d/asterisk_mbox.sh:/etc/cont-init.d/asterisk_mbox.sh:ro
+
+  # To test the add-on against an HA instance
+  home-assistant:
+    container_name: home-assistant
+    image: homeassistant/home-assistant:2023.2
```

```suggestion
    image: homeassistant/home-assistant:stable
```

I don't want to have to keep this updated.